doc/manual/plugins_local/pick.rst

```diff
@@ -3,255 +3,105 @@
-   first one found, using the peak's coordinates.
-3) In Quick Mode OFF, it will search the area for all peaks and
-   evaluate the peaks based on the criteria in the "Settings" tab of the UI
-   (see "The Settings Tab", below) and try to locate the best candidate
-   matching the settings.
-
-If a candidate is found
-^^^^^^^^^^^^^^^^^^^^^^^
-
-The candidate will be marked with a `Point` (usually an "X") in the
-channel viewer canvas, centered on the object as determined by the
-horizontal and vertical FWHM measurements.
-
-The top set of tabs in the UI will be populated as follows.
-The "Image" tag will show the contents of the cutout area:
+**Figures to show specific plugin usage**
 
-  .. image:: figures/pick-cutout.png
-     :width: 400px
-     :align: center
-
-The widget in this tab is a Ginga widget and so can be zoomed and panned
-with the usual keyboard and mouse bindings (e.g. scroll wheel).  It will
-also be marked with a `Point` centered on the object and additionally the
-pan position will be set to the found center.
-
-The "Contour" tab will show a contour plot:
-
-  .. image:: figures/pick-contour.png
-     :width: 400px
-     :align: center
-
-This is a contour plot of the area immediately surrounding the
-candidate, and not usually encompassing the entire region of the pick
-area.  You can use the vertical slider to the right of the plot to
-increase or decrease the area of the contour plot.
-
-The "FWHM" tab will show a FWHM plot:
-
-  .. image:: figures/pick-fwhm.png
-     :width: 400px
-     :align: center
-
-The blue lines show measurements in the X direction and the green lines
-show measurements in the Y direction.  The solid lines indicate actual
-pixel values and the dotted lines indicate the fitted 1D gaussians.
-The shaded green and blue regions indicate the FWHM measurements.
-(Moffat fitting is also available.)
-
-The "Radial" tab contains a radial profile plot:
-
-  .. image:: figures/pick-radial.png
-     :width: 400px
-     :align: center
-
-Plotted points in blue are data values, and a line is fitted to the
-data.
-
-The "Cuts" tab contains a profile plot for the vertical and horizontal
-cuts represented by the crosshairs present in Quick Mode ON.  This plot
-is updated in real time as the pick area is moved.  In Quick Mode OFF
-this plot is not updated.  In the figure below, you can see the addition
-of the crosshair to the pick region, and the associated plot in the Cuts
-tab.
+.. figure:: figures/pick-move-draw-edit.png
+   :width: 400px
+   :align: center
+   :alt: Defining a Pick area
 
-  .. image:: figures/pick-cuts.png
-     :width: 800px
-     :align: center
+   Defining a ``Pick`` area.
 
-The "Readout" tab will be populated with a summary of the measurements:
+.. figure:: figures/pick-cutout.png
+   :width: 400px
+   :align: center
+   :alt: Image tab of Pick area
 
-  .. image:: figures/pick-readout.png
-     :width: 400px
-     :align: center
```

Comment:
You are looking at the removed code (red), which does not correspond to the added text (green) directly below. Sorry for the confusing diff. Maybe for this file, it makes more sense if you view it directly (the "View" button) instead of looking at the diff.